v1.0/known-limitations.md

```diff
@@ -28,6 +28,8 @@ Within a single [transaction](transactions.html):
 - A [`CREATE TABLE`](create-table.html) statement containing [`FOREIGN KEY`](foreign-key.html) or [`INTERLEAVE`](interleave-in-parent.html) clauses cannot be followed by statements that reference the new table.
 - A table cannot be dropped and then recreated with the same name. This is not possible within a single transaction because `DROP TABLE` does not immediately drop the name of the table. As a workaround, split the [`DROP TABLE`](drop-table.html) and [`CREATE TABLE`](create-table.html) statements into separate transactions.
 
+    Note that this limitation also applies to [`TRUNCATE`](../v1.0/truncate.html), which implicitly drops and recreates a table.
+
```

Comment:
Actually this limitation needs to be placed after the CREATE TABLE limitation. TRUNCATE drops and then creates a table, and an INSERT will fail only because the CREATE table has an FK

Reply:
Hmm, I guess I'm not being clear here. Since `truncate` is equivalent to dropping and recreating a table with the same name, I'm saying that the last bullet also means `truncate` can't be used in a transaction. 

Can you clarify how this connects to create table? I have a feeling your referencing another limitation that we haven't yet documented...

Reply:
TRUNCATE can be used in a transaction, but since it creates a table it suffers from the limitations of CREATE. All limitations of CREATE apply to it.

Reply:
Updated. PTAL.

Reply:
you need to update this.

Reply:
Done.